Builds failing for Android and Linux 2024-10-02

I'm surprised V8 keeps having build issues this often. We're supposedly using the Chromium stable version.

Android is failing on

```
In file included from ../../v8/third_party/abseil-cpp/absl/base/internal/cycleclock.cc:26:
In file included from /usr/lib/gcc/x86_64-linux-gnu/14/../../../../include/c++/14/chrono:3360:
In file included from /usr/lib/gcc/x86_[64](https://github.com/tommie/v8go/actions/runs/11136183688/job/30947620891#step:12:65)-linux-gnu/14/../../../../include/c++/14/bits/chrono_io.h:39:
In file included from /usr/lib/gcc/x86_64-linux-gnu/14/../../../../include/c++/14/format:58:
/usr/lib/gcc/x86_64-linux-gnu/14/../../../../include/c++/14/bits/unicode.h:807:4: error: expression is not assignable
```

Linux is failing on

```
ERROR at //build/config/linux/pkg_config.gni:104:17: Script returned non-zero exit code.
    pkgresult = exec_script(pkg_config_script, args, "json")
                ^----------
Current dir: /home/runner/work/v8go/v8go/deps/.build/linux_amd64/
Command: python3 /home/runner/work/v8go/v8go/deps/v8/build/config/linux/pkg-config.py glib-2.0 gmodule-2.0 gobject-2.0 gthread-2.0
Returned 1.
stderr:

Package glib-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `glib-2.0.pc'
to the PKG_CONFIG_PATH environment variable
Package 'glib-2.0', required by 'virtual:world', not found
Package 'gmodule-2.0', required by 'virtual:world', not found
Package 'gobject-2.0', required by 'virtual:world', not found
Package 'gthread-2.0', required by 'virtual:world', not found
Could not run pkg-config.
```

--- a/deps/build.py
+++ b/deps/build.py
@@ -76,6 +76,7 @@ v8_target_cpu="%s"
 clang_use_chrome_plugins=false
 use_custom_libcxx=false
 use_sysroot=false
+use_glib=false
 symbol_level=%s
 strip_debug_info=%s
 is_component_build=false

